### PR TITLE
feat(modals): use dark theme

### PR DESCRIPTION
**Pull request by virtuacoplenny**
Getting dark modals to work did require hacks that may or may not be acceptable, so I'm opening this PR to get opinions.

~~1. For some reason, currently unknown to me, children of Atlaskit's modal have their i18n context value set to undefined. The workaround was to wrap the children of Atlaskit modals in the I18nextProvider.  The question becomes if this is an acceptable workaround for now. I had difficulty tracking down the cause of the issue through the react-lib-laden stack trace. I will be giving it at least one more debug session, though.
2. Because of the workaround of rewrapping modal children in I18nextProvider, the passing around of i18next to the Device Selection Popup was removed because the popup imports i18n now.~~~~ I've used atlastkit's withContextFromProps to push i18n down into the dialog children. This doesn't seem that different from my initial workaround but it's the "official" workaround so I'm going with it.
3. atlaskit looks to have started using flow and some of the .flow files were causing errors. I set those files to be ignored.
4. I modified some of css/themes/_light.scss, specifically modals and buttons, to support all modals having a dark appearance. This makes the concept "light.scss" seem incorrect. EDIT: Yana said this is fine.

Other quirkier changes:
- Changed DeviceSelection css to support dark theme. DeviceSelection was written during earlier atlaskit development and had some styling-related hacks that needed to be adjusted. DeviceSelection's styling hacks can be re-evaluated in the near future as it has a slight redesign planned.
- FeedbackDialog's textarea was swapped with atlaskit's field text area so dark theme could be applied automatically.

### CODE REVIEW

**Comment by hristoterezov** [comment]
I've just tested the PR and for some reason the device selection dialog is not showing for me. 

Is it working for you?

**Comment by hristoterezov** [comment]
When I click the "Shared Video" button I see a black non-transparent background for a few seconds and after that the background is changed to black transparent one. Is this a bug?

**Comment by virtuacoplenny** [comment]
> When I click the "Shared Video" button I see a black non-transparent background for a few seconds and after that the background is changed to black transparent one. Is this a bug?

I see that on beta right now. Do you see it on beta?

**Comment by hristoterezov** [comment]
Yes! Sorry for the noise!

**Comment by hristoterezov** [comment]
> I've just tested the PR and for some reason the device selection dialog is not showing for me.
> 
> Is it working for you?

I've rebased and rebuild and it started working! @virtuacoplenny Sorry for the noise !

**Comment by virtuacoplenny** [comment]
Let me know if your package-lock changes. I did most of this development from my home mac which runs and older version of npm. I noticed just now a bunch of integrity shas changed while trying to switch back onto the dark modals branch.

**Comment by hristoterezov** [comment]
Yes. The package-lock.json changes with bunch of integrity shas.

File: react/features/base/dialog/components/StatelessDialog.web.js [hunk 1]
```diff
@@ -112,36 +125,86 @@ class StatelessDialog extends Component<Props> {
...
+            titleString,
+            titleKey,
+            width
+        } = this.props;
 
-            /**
-             * Enabled light theme for dialogs until all in-dialog components
-             * support dark theme.
-             */
-            <AtlasKitThemeProvider mode = 'light'>
```
Comment (hristoterezov) [inline react/features/base/dialog/components/StatelessDialog.web.js]: @virtuacoplenny I think the device selection dialog for filmstrip only mode is not looking correct. Maybe is related to this change.

Comment (hristoterezov) [reply react/features/base/dialog/components/StatelessDialog.web.js]: Ignore this! It is working!

**Pull request merged**